molecule/default/molecule.yml

```diff
@@ -0,0 +1,26 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+  # name: podman
+platforms:
+  - name: instance
+    image: "geerlingguy/docker-${MOLECULE_DISTRO:-fedora35}-ansible:latest"
```

Comment:
Need to add "docker.io/" to the image so podman doesn't complain about the various alternate sources.